This works around:

./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: warning: asm operand 2 probably doesn't match constraints
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'
./libavutil/arm/intmath.h:81:5: error: impossible constraint in 'asm'


--- ffmpeg-0.8.7/libavutil/arm/intmath.h	2011-11-22 14:17:36.466615233 -0500
+++ ffmpeg-0.8.7/libavutil/arm/intmath.h	2011-11-22 14:08:48.737864995 -0500
@@ -74,6 +74,7 @@
     return x;
 }
 
+/*
 #define av_clip_uintp2 av_clip_uintp2_arm
 static av_always_inline av_const unsigned av_clip_uintp2_arm(int a, int p)
 {
@@ -81,6 +82,7 @@
     __asm__ ("usat %0, %2, %1" : "=r"(x) : "r"(a), "i"(p));
     return x;
 }
+*/
 
 
 #else /* HAVE_ARMV6 */
